srcpkgs/ktorrent/template

```diff
@@ -1,20 +1,26 @@
 # Template file for 'ktorrent'
 pkgname=ktorrent
-version=5.2.0
+version=20.12.1
 revision=1
 build_style=cmake
+configure_args="-DWITH_SYSTEM_GEOIP=ON"
 hostmakedepends="extra-cmake-modules qt5-host-tools qt5-qmake gettext"
 makedepends="qt5-devel qt5-script-devel ki18n-devel kio-devel kxmlgui-devel
  knotifications-devel ksolid-devel kconfig-devel kcoreaddons-devel kdoctools-devel
  kross-devel kcmutils-devel sonnet-devel kdbusaddons-devel kcrash-devel
  karchive-devel knotifyconfig-devel libktorrent-devel kplotting-devel
- gmp-devel taglib-devel kdnssd-devel kdewebkit-devel"
+ gmp-devel taglib-devel kdnssd-devel kdewebkit-devel geoip-devel syndication-devel"
+depends="geoip-data"
 short_desc="BitTorrent client based on the KDE platform"
 maintainer="Norbert Vegh <vegh@norvegh.com>"
 license="GPL-2.0-or-later"
 homepage="https://www.kde.org/applications/internet/ktorrent"
-distfiles="${KDE_SITE}/ktorrent/${version}/ktorrent-${version}.tar.xz"
-checksum=d1031d745c1ace02d11b59c426610aee779d208e3d1ae81f8559c265173732ff
+distfiles="https://github.com/KDE/ktorrent/archive/v${version}.tar.gz"
```

Comment:
Use `KDE_SITE/release-service` instead, also for libktorrent.

Reply:
> Use `KDE_SITE/release-service` instead, also for libktorrent.

Thank you, will do!

Reply:
Done!